Add validation for page number (can't be 0)

Passing `page=0` causes a server error when it should throw validation.

Examples:

https://api.open.fec.gov/v1/committees/?api_key=DEMO_KEY&committee_id=C00431445&page=0

https://api.open.fec.gov/v1/candidates/?sort_nulls_last=false&sort_hide_null=false&office=P&per_page=100&sort_null_only=false&sort=name&page=0&election_year=2024&api_key=DEMO_KEY

```
   2020-07-20T10:41:24.80-0400 [APP/PROC/WEB/4] ERR [2020-07-20 14:41:24 +0000] [301] [ERROR] Error handling request /v1/candidates/?sort_nulls_last=false&sort_hide_null=false&office=P&per_page=100&sort_null_only=false&sort=name&page=0&election_year=2024
   2020-07-20T10:41:24.80-0400 [APP/PROC/WEB/4] ERR Traceback (most recent call last):
   2020-07-20T10:41:24.80-0400 [APP/PROC/WEB/4] ERR   File "/home/vcap/deps/0/python/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1244, in _execute_context
   2020-07-20T10:41:24.80-0400 [APP/PROC/WEB/4] ERR     cursor, statement, parameters, context
   2020-07-20T10:41:24.80-0400 [APP/PROC/WEB/4] ERR   File "/home/vcap/deps/0/python/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 552, in do_execute
   2020-07-20T10:41:24.80-0400 [APP/PROC/WEB/4] ERR     cursor.execute(statement, parameters)
   2020-07-20T10:41:24.80-0400 [APP/PROC/WEB/4] ERR psycopg2.errors.InvalidRowCountInResultOffsetClause: OFFSET must not be negative
```

--- a/tests/test_candidates.py
+++ b/tests/test_candidates.py
@@ -217,6 +217,12 @@ class CandidateFormatTest(ApiBaseTest):
         )
         self.assertEqual([each['candidate_id'] for each in results], ['2', '3', '1'])
 
+    def test_page_validation(self):
+        response = self.app.get(
+            api.url_for(CandidateList, page=0)
+        )
+        self.assertEqual(response.status_code, 422)
+
 
 class TestCandidateHistory(ApiBaseTest):
     def setUp(self):

--- a/webservices/args.py
+++ b/webservices/args.py
@@ -10,9 +10,9 @@ from webservices.common.models import db
 
 
 def _validate_natural(value):
-    if value < 0:
+    if value <= 0:
         raise exceptions.ApiError(
-            'Must be a natural number',
+            'Must be greater than zero',
             status_code=422
         )
 

